backend/estimate-service/src/main/java/org/egov/works/util/EstimateServiceConstant.java

```diff
@@ -19,4 +21,13 @@ public class EstimateServiceConstant {
     public static final String MASTER_FUNCTIONS = "Functions";
     public static final String MASTER_BUDGET_HEAD = "BudgetHead";
     public static final String MASTER_SCHEME = "Scheme";
+
+    public static final String MASTER_LOCATION = "TenantBoundary";
+
+    //work flow
+    public static final String ESTIMATE_MODULE_NAME = "estimate";
+
+    //location
+    public static final String BOUNDARY_ADMIN_HEIRARCHY_CODE = "ADMIN";
```

Comment:
Keep location related constants closeby.

Reply:
The same approach is followed.